### ISSUE TITLE
Top-level folder in nvim-tree doesn't have enough color contrast

### ISSUE THREAD

**Issue by mrjones2014** [Pocco81/Catppuccino.nvim#2]
![image](https://user-images.githubusercontent.com/8648891/130222916-19151c1b-2d24-4551-8923-4c2aafbbb9a1.png)

it's difficult to read.

**Comment by Pocco81** [comment]
Hey! So comming from an NvChad background the top level folder was "transparent" (aka same color as the background) and you were only able to see it while hovering over.

I really liked that and kept it here as you can see in the screenshots. Do you like it? Do you think it should have a solid color?

**Comment by Pocco81** [comment]
Btw, not sure if you've been reading the commit messages, but I recently finished all of the integrations and the current settings can be changed like so:

```
require("catppuccino").setup({
colorscheme = "catppuccino",
    transparency = false,
	styles = {
		comments = "italic",
		fuctions = "italic",
		keywords = "italic",
		strings = "NONE",
		variables = "NONE",
	},
	integrations = {
		treesitter = true,
		native_lsp = {
			enabled = true,
			styles = {
				errors = "italic",
				hints = "italic",
				warnings = "italic",
				information = "italic"
			}
		},
		lsp_trouble = false,
		lsp_saga = false,
		gitgutter = false,
		gitsigns = false,
		telescope = false,
		nvimtree = false,
		which_key = false,
		indent_blankline = false,
		dashboard = false,
		neogit = false,
		vim_sneak = false,
		fern = false,
		barbar = false,
		bufferline = false,
		markdown = false,
	}
})
```

**Comment by mrjones2014** [comment]
I have the nvim-tree integration enabled.

**Comment by Pocco81** [comment]
And do you the transparent nvim tree? You are using dev branch right?

**Comment by mrjones2014** [comment]
No I'm using master branch. It's not transparent for me but the light purple in the screenshot.

**Comment by Pocco81** [comment]
Ok the feature was merged and you should be able to use it now:

### NvimTree

![image](https://user-images.githubusercontent.com/58336662/130275916-7d433950-9f7d-40c3-849e-51a858516f47.png)

 Cheers :+1:

**Comment by mrjones2014** [comment]
I'm getting this error when updating with Packer:

![image](https://user-images.githubusercontent.com/8648891/130276354-0c8ca52a-0435-4d09-97ab-ba52f6c975b6.png)

**Comment by Pocco81** [comment]
I haven't done any resets, or really anything to break the history. Perhaps you tried to pull from `dev` while having `main` and resulted in a conflict? If it keeps failing just remove it manually `rm -r ~/.local/share/nvim/site/pack/packer/start/Catppuccino.nvim`.

**Comment by mrjones2014** [comment]
that fixed it, thanks.
